docs/abi.rst

```diff
@@ -210,7 +210,7 @@ PyTeal Type                                    ARC-4 Type             Dynamic /
 :any:`abi.Address`                             :code:`address`        Static                              A 32-byte Algorand address. This is an alias for :code:`abi.StaticArray[abi.Byte, Literal[32]]`.
 :any:`abi.DynamicArray[T] <abi.DynamicArray>`  :code:`T[]`            Dynamic                             A variable-length array of :code:`T`
 :any:`abi.String`                              :code:`string`         Dynamic                             A variable-length byte array assumed to contain UTF-8 encoded content. This is an alias for :code:`abi.DynamicArray[abi.Byte]`.
-:any:`abi.Tuple`\*                             :code:`(...)`          Static when all elements are static A tuple of multiple types
+:any:`abi.Tuple`\*, :any:`abi.NamedTuple`\*    :code:`(...)`          Static when all elements are static A tuple of multiple types
```

Comment:
I'd like to suggest removing the asterisk on NamedTuple on this line and at the beginning of "While we are still on PyTeal 3.10..." below. Having two of the same asterisk symbols doesn't make sense to me.
```suggestion
:any:`abi.Tuple`\*, :any:`abi.NamedTuple`      :code:`(...)`          Static when all elements are static A tuple of multiple types
```